doc/themes/kconfig/versions.html

```diff
@@ -0,0 +1,21 @@
+{# Add rst-badge after rst-versions for small badge style. #}
+  <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
+    <span class="rst-current-version" data-toggle="rst-current-version">
+      <span class="fa fa-book">Kconfig Reference</span>
```

Comment:
```suggestion
      <span class="fa fa-book"> Kconfig Reference</span>
```

Reply:
Guessing it was meant to lowercase the `R`, so fixed that. :)